packages/app/app/stacks/stacks-orders.tsx

```diff
@@ -27,8 +28,9 @@ export const StackOrders = ({ chainId, address }: StackOrdersProps) => {
   );
   const activeOrders = filterActiveOrders(currentStackOrders);
   const completedOrders = filterCompletedOrders(currentStackOrders);
+  const cancelledOrders = filterCancelledOrders(currentStackOrders);
 
-  useEffect(() => {
+  const fetchData = useCallback(() => {
```

Comment:
`fetchOrders` seems more accurate to me

Reply:
Also could be `fetchStacks`

Reply:
Makes sense, I'll change it to `fetchStacks`

Reply:
fixed: https://github.com/SwaprHQ/stackly-ui/pull/40/commits/412a3d165642b1e57ba9cc7eed23943c9134b12c